src/nft/components/profile/view/FilterSidebar.tsx

```diff
@@ -84,8 +79,8 @@ export const FilterSidebar = ({
   return (
     // @ts-ignore
     <AnimatedBox
-      position={{ sm: 'fixed', md: 'sticky' }}
-      top={{ sm: '40', md: 'unset' }}
+      position="sticky"
+      top="0"
```

Comment:
I think we need `72` here so it doesn't get hidden under the navbar

Reply:
sg done